### PR TITLE
Fix system test runner commit sha

### PR DESCRIPTION
**Pull request by TonyCTHsu**
**What does this PR do?**

I figured out that the commit SHA must be on existing master branch, since the previous sha fails after PR merged/branch deleted.

### CODE REVIEW

File: .github/workflows/lib-injection.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
     steps:
       - name: lib-injection test runner
         id: lib-injection-test-runner
-        uses: DataDog/system-tests/lib-injection/runner@1af3241d5b6a928199528a8cbfc5698564f5d260
+        uses: DataDog/system-tests/lib-injection/runner@4eb1ec01b61c1c7e84f219127a91d07f03b0f85e
```
Comment (ivoanjo) [inline .github/workflows/lib-injection.yml:54]: Is there any way to get the "latest", or "master"? Or would we want to explicitly always use an older version here?

(Also, would it make sense to leave a comment here explaining how to update to the latest version?)

**Review by ivoanjo — APPROVED** [review]
:+1: Thanks for looking into this!

File: .github/workflows/lib-injection.yml (hunk 1 above)
Comment (GustavoCaso) [reply .github/workflows/lib-injection.yml:54]: I was going to suggest always pointing to `master|main`. Could we do that?

**Review by GustavoCaso — APPROVED** [review]

File: .github/workflows/lib-injection.yml (hunk 1 above)
Comment (TonyCTHsu) [reply .github/workflows/lib-injection.yml:54]: Nope. It is a SHA.

Comment (ivoanjo) [reply .github/workflows/lib-injection.yml:54]: Maybe we could use https://github.com/marketplace/actions/latest-tag ? That seems exactly what we're missing here.

**Pull request merged**